### DISCUSSION TITLE
extending flet using flutter

### DISCUSSION THREAD

**Discussion by bitmvk** [flet-dev/flet#3211] [Q&A]
### Question

hey,
today i tried for extending flet using flutter. After checking the `flet_video` package i found out that we need to edit the `main.dart` file in the flet project. I did the editing required by using [contribution docs](https://github.com/flet-dev/flet/blob/main/CONTRIBUTING.md), and it worked on desktop.

How to get this to work in android?

@ndonkoHenri @FeodorFitsner (sorry in advance for tagging)

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
- Replace "localhost" [here](https://github.com/flet-dev/flet/blob/main/client/lib/main.dart#L38) by your ip address
- open/turn-on your android emulator device
- do `flutter run` in your terminal and select the emulator device
